src/components/events-list/event-summary.js

```diff
@@ -33,11 +33,14 @@ const EventSummary = ({ event }) => (
       />
     </Table.Cell>
     <Table.Cell>
-      <StatusBadge type={isWarningEvent(event) ? 'warning' : 'success'}>
-        {event?.type}
-      </StatusBadge>
+      <>
+        <StatusBadge type={isWarningEvent(event) ? 'warning' : 'success'}>
+          {event?.type}
+        </StatusBadge>
+        <WarningState event={event}></WarningState>
+      </>
```

Comment:
Add a class here with the following styling to add spacing between the StatusBadges:
```
.events_table .type {
  display: grid;
  gap: var(--eds_spacing_small);
  padding: var(--eds_spacing_small) 0;
}
```
```suggestion
      <div className="type">
        <StatusBadge type={isWarningEvent(event) ? 'warning' : 'success'}>
          {event?.type}
        </StatusBadge>
        <WarningState event={event}></WarningState>
      </div>
```

Reply:
Fixed